src/App.js

```diff
@@ -1,17 +1,45 @@
-import React from 'react';
-import '../src/css/style.css';
-import Navbar from './pages/Navbar';
-import LoginPage from './components/LoginPage';
-import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
-
+import React, { useEffect } from 'react';
+import FinalLayout from './pages/FinalLayout/FinalLayout';
+import CreateAccount from '../src/pages/SignupPage/CreateAccount';
+import LoginPage from '../src/pages/LoginPage/LoginPage';
+import { BrowserRouter as Router, Routes, Route, Link } from "react-router-dom";
+import ProfilePage from './pages/ProfilePage/ProfilePage';
+import FeedPage from '../src/pages/FeedPage/FeedPage';
+import ViewPost from './components/ViewPost/ViewPost';
+import FeedLayout from '../src/components/FeedLayout';
+import { Posts } from './components/post/Posts';
+import Comments from './components/simple-comments/Comments';
+import CommentWindow from './components/simple-comments/ CommentWindow';
 function App() {
+
     return (
-        <Router>
-          {/* <Navbar /> */}
-          <Routes>
-            <Route exact path="/userlogin" element={<LoginPage />} />
-          </Routes>
-        </Router>
+        <>
+            <Router>
```

Comment:
create another component for routers and its routes, and use that component here